src/moj/components/header/_header.scss

```diff
@@ -31,7 +31,7 @@
 
 }
 
-.moj-header__logotype-crest {
+.moj-header__logotype-crest-2024 {
```

Comment:
we don't need to change the css class name

Reply:
Noted and amended.